core/templates/dev/head/pages/exploration-editor-page/editor-navigation/editor-navigation.directive.html

```diff
@@ -17,18 +17,156 @@
     border-radius: 10%;
     bottom: 5px;
     height: 15px;
-    position: absolute;
     right: 3px;
+    position: absolute;
     text-align: center;
   }
   editor-navigation > ul > li.nav-link {
     color: white;
     text-align: center;
     width: 46px;
-}
+  }
+  .oppia-dropdown {
+    border-bottom: 0;
+    border-left: 0;
+    border-right: 0;
+    border-top: 0;
+    max-width: 280px;
+    min-width: 170px;
+    padding: 8px 0;
+  }
+  .oppia-dropdown .oppia-dropdown-item {
+    height: 48px;
+    margin-right: 0;
+    position: relative;
+    padding: 0 16px;
+  }
+  .oppia-dropdown-toggle {
+    border: none;
+    margin-top: 8px;
+  }
+  .oppia-dropdown-item .oppia-dropdown-element {
+    color: #000;
+    margin: 0;
+    -ms-transform: translateY(-50%);
+    position: absolute;
+    text-decoration: none;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+  .oppia-dropdown-item .oppia-dropdown-element .oppia-icons {
+    padding-right: 10px;
+  }
+  .oppia-dropdown-item .oppia-dropdown-element .oppia-dropdown-inner {
+    font-family: 'Roboto';
+  }
+  .oppia-dropdown-item .oppia-editor-warnings-indicator {
+    margin: 0;
+    -ms-transform: translateY(-50%);
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+  }
 </style>
 <ul class="nav navbar-nav oppia-navbar-nav ng-cloak">
-  <li ng-class="{'active': getActiveTabName() === 'main', 'uib-dropdown': countWarnings()}" ng-attr-uib-dropdown="<[countWarnings()]>" class="nav-item">
+  <li class="nav-item dropdown d-block d-lg-none">
+    <a class="btn dropdown-toggle oppia-dropdown-toggle text-white" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+      Options
+    </a>
+    <ul class="dropdown-menu oppia-dropdown" aria-labelledby="dropdownMenuButton">
```

Comment:
Change this to `oppia-exploration-editor-tabs-dropdown. (Similar changes to the name of the `li` class maybe?)